cuda_core/cuda/core/experimental/_launcher.py

```diff
@@ -134,6 +146,12 @@ def launch(kernel, config, *kernel_args):
         drv_cfg.hStream = config.stream.handle
         drv_cfg.sharedMemBytes = config.shmem_size
         drv_cfg.numAttrs = 0  # TODO
+        if config.cluster:
+            drv_cfg.numAttrs += 1
+            attr = cuda.CUlaunchAttribute()
+            attr.id = cuda.CUlaunchAttributeID.CU_LAUNCH_ATTRIBUTE_CLUSTER_DIMENSION
+            attr.value.clusterDim.x, attr.value.clusterDim.y, attr.value.clusterDim.z = config.cluster
+            drv_cfg.attrs = [attr]  # TODO: WHAT!!
```

Comment:
@vzhurba01 I noticed the binding of `CUlaunchConfig` is being weird
- I cannot just do `cfg.attrs.append(attr)` here
- I cannot empty initialize it with `cfg.attrs = []` and then do `cfg.attrs.append(attr)` either

Reply:
If I don't do this I'd get
```
cuda.core.experimental._utils.CUDAError: CUDA_ERROR_INVALID_VALUE: invalid argument
```
when calling `cuLaunchKernelEx`.

Reply:
> https://github.com/NVIDIA/cuda-python/issues/262

Created an issue to track a documentation improvement. From offline talk this is because the struct getters return values by copy. This limitation comes from the bindings needed to transform Python values/formats into C values/formats.

Reply:
Thanks, Vlad! Cleaned up this part in commit 895c9fa.